pages/api/policies.js

```diff
@@ -60,19 +58,18 @@ export default async (req, res) => {
         mapToClientModel(policy)
       );
 
-      res.status(StatusCodes.OK).json({
+      return res.status(StatusCodes.OK).json({
         data: policies,
         pageToken: listPoliciesResponse.nextPageToken,
       });
     } catch (error) {
       console.error("Error listing policies", error);
 
-      res
+      return res
         .status(StatusCodes.INTERNAL_SERVER_ERROR)
         .json({ error: ReasonPhrases.INTERNAL_SERVER_ERROR });
     }
   }
-
   if (req.method === "POST") {
```

Comment:
After adding `return` on line 68, coverage tells me that this statement isn't covered but I can't figure out how else to test this. There's already tests that cover unimplemented methods that I thought would catch the other side of this if.

Reply:
I think the implicit else there can never be reached because of that check for unimplemented methods. With the `return` I think the `if` could be dropped since it can't have any other value than `POST` by that point. 

But if you want to revert adding the `return` statement that works, too.

Reply:
Thank you, Alex, you always know what's up. I ended up removing the `if` check around the `POST` sections.